ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -0,0 +1,47 @@
+from typing import Callable, TypeVar, Union
+from durabletask import task
+from datetime import datetime
+from dapr.ext.workflow.workflow_context import WorkflowContext, Workflow
+
+from dapr.ext.workflow.workflow_activity_context import WorkflowActivityContext
+
+T = TypeVar('T')
+TInput = TypeVar('TInput')
+TOutput = TypeVar('TOutput')
+
+
+class DaprWorkflowContext(WorkflowContext):
```

Comment:
One important API we're missing is `wait_for_external_event`. We got tons of asks for this after v1.10, so I think it's a good idea to include it in the first version of the Python SDK.

Reply:
Included `wait_for_external_event` and `continue_as_new`